app-text/robodoc/robodoc-4.99.36.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
```

Comment:
This change seems useless, and you can not commit ebuilds with EAPI5. (Use repoman to commit) Also note what I said about removing this version after the bump to latest gets done.

Reply:
I was just going with the (seeming) flow of things. ebuild for this version got kicked out anyway as you noted